docs/backplane/guideline.md

```diff
@@ -0,0 +1,103 @@
+# Goals
+
+Answer the following questions:
+* How can LPSRE team members proactively address backplane permissions concerns to facilitate the review process?
+* What conventions should be followed to make the intention behind changes more obvious?
+* Review comments are inconsistent (though not necessarily conflicting) between reviewers of backplane permissions changes. How can that be improved?
+* How can we reconcile SRE-P expectations with LPSRE expectations for what constitutes reasonable permissions?
+
+# Requirements
+
+Backplane governs Red Hat access to both the Managed OpenShift cluster and the cloud account on which the cluster is hosted.  Permissions for each product are documented in the respective service definitions here:
+* https://docs.openshift.com/dedicated/osd_architecture/osd_policy/policy-process-security.html#privileged-access_policy-process-security 
+* https://docs.openshift.com/rosa/rosa_architecture/rosa_policy_service_definition/rosa-policy-process-security.html#rosa-policy-rh-access_rosa-policy-process-security
+
+For the purposes of this document we are focusing on the in-cluster access, not the cloud account access. Since these are currently the same for OSD and ROSA (May 15, 2023) we can focus on a single set of requirements.  The following is the a summary taken from the ROSA documents and specifies only read or write operations that are something other than `None`:
```

Comment:
@jewzaam are we OK with not documenting the cloud account access for now?

Also, I don't like the fact that this is expressed as a snapshot rather than an accepted pattern.

Reply:
FWIW cloud access is not managed through MCC. It's configured in `app-interface` so even if we discuss it here we should redirect to the authoritative document which should be located near the backplane config in `app-interface itself.

Reply:
I believe this is what Naveen means by requirements doc https://github.com/openshift/managed-cluster-config/pull/1678/files#diff-a68aaa6a55aef8a6f631e8ecdd46c50af8bceffd8e6cec11c0c571d8e95e3540

Reply:
Cloud access needs to be documented.  I did include this for the draft SREP requirements.  This should change.

```suggestion
For the purposes of this document we are focusing on all backplane access to a cluster, including in-cluster RBAC and cloud account access.  The following is the a summary taken from the ROSA documents and specifies only read or write operations that are something other than `None`:
```

And the bullet points following updated.